packages/fx-core/src/component/driver/tools/installDriver.ts

```diff
@@ -13,11 +14,13 @@ import { InvalidParameterUserError } from "./error/invalidParameterUserError";
 import { InstallToolArgs } from "./interfaces/InstallToolArgs";
 import { DepsManager, DepsType, EmptyLogger, EmptyTelemetry } from "../../../common/deps-checker";
 import { FuncInstallationUserError } from "./error/funcInstallationUserError";
+import { DotnetInstallationUserError } from "./error/dotnetInstallationUserError";
 
 const ACTION_NAME = "tools/install";
 const outputName = Object.freeze({
   SSL_CRT_FILE: "SSL_CRT_FILE",
   SSL_KEY_FILE: "SSL_KEY_FILE",
+  DOTNET_PATH: "DOTNET_PATH",
```

Comment:
Why output this? It's a bit weird there's only dotnet path but no func path.

Reply:
The dotnet/command driver need the path as input.

The func path is never used in the other drivers.

Reply:
Just to keep UX consistency - output `FUNC_PATH` as well when improving func install later.